src/core/OWNERS

```diff
@@ -0,0 +1,14 @@
+// For an explanation of the OWNERS rules and syntax, see:
+// https://github.com/ampproject/amp-github-apps/blob/master/owners/OWNERS.example
+
+{
+  rules: [
+    {
+      owners: [
+        {name: 'dvoytenko'},
+        {name: 'jridgewell'},
```

Comment:
Sorry, isn't there a key that says I should be added to every review?

PRs into this directory have to be done carefully to ensure the code can be DCEd.

Reply:
There is now!